tests/vmcheck/test-layering-relayer.sh

```diff
@@ -84,9 +84,30 @@ echo "ok pkg foo relayered on deploy"
 # REBASE
 
 commit=$(vm_cmd ostree commit -b rebase_test --tree=ref=vmcheck)
-vm_rpmostree rebase rebase_test
+vm_rpmostree rebase --skip-purge rebase_test
 reboot_and_assert_base $commit
 echo "ok rebase"
 
 vm_assert_layered_pkg foo present
 echo "ok pkg foo relayered on rebase"
+
+# rollup: install/deploy/uninstall
+
+assert_status_jq ".deployments[0][\"base-checksum\"] == \"${commit}\"" \
+                 '.deployments[0]["packages"]|index("foo") >= 0' \
+                 '.deployments[0]["packages"]|index("nonrootcap")|not'
+vm_rpmostree install nonrootcap
+assert_status_jq ".deployments[0][\"base-checksum\"] == \"${commit}\"" \
+                 '.deployments[0]["packages"]|index("foo") >= 0' \
```

Comment:
Doesn't do what you'd think:
```
echo '["foo", "bar"]' | jq 'has("foo")'
jq: error (at <stdin>:1): Cannot check whether array has a string key
```

I looked but didn't see a jq operator for searching an array.